main.py

```diff
@@ -67,3 +68,34 @@ def power_on(settings: Annotated[Settings, Depends(get_settings)]):
         '<network-data xmlns="http://www.Bose.com/Schemas/2012-12/NetworkMonitor/" />'
         "</network-landscape></diagnostic-data></device-data>"
     )
+
+@app.get("/bmx/registry/v1/services",
+         tags=["bmx"]
+)
+def bmx_services(settings: Annotated[Settings, Depends(get_settings)]):
+    response = BmxResponse()
+    response._links = dict(bmx_services_availability = dict(href = '../servicesAvailability'))
+    response.askAgainAfter = 1277728
+    tunein = Service()
+    tunein._links = dict(bmx_navigate=Link(href="/v1/navigate"), bmx_token = Link(href="/v1/token"),
+        self = Link(href="/"))
```

Comment:
```suggestion
    tunein._links = {'bmx_navigate': '/v1/navigate', 'bmx_token': '/v1/token', 'self': '/'}
```

or you'll have to add `.href` after every `Link`.